packages/edit-post/src/editor.js

```diff
@@ -34,14 +33,12 @@ function Editor( {
 	initialEdits,
 	...props
 } ) {
-	const isLargeViewport = useViewportMatch( 'medium' );
```

Comment:
We don't need this check any more?

Reply:
Good catch. The later `! isLargeViewport` check does toggle the fixed toolbar setting on for mobile devices. So we should find a way to preserve it or architect it differently.